### ISSUE TITLE
Upgrading to Node 6.8.0 gives "TypeError: Cannot call a class as a function"

### ISSUE THREAD

**Issue by perhallstroem** [rapid7/le_node#139]
After upgrading to Node 6.8.0, a TypeError is thrown when starting the application. The message is "Cannot call a class as a function". 

```
TypeError: Cannot call a class as a function
    at exports.default (<our project>/node_modules/babel-runtime/helpers/classCallCheck.js:7:11)
    at new Logger (<our project>/node_modules/le_node/lib/logger.js:146:34)
... our code ...
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
```

Without any code changes, this happens when using 6.8.0 but not 6.7.0. We are, as far as I can tell, not "calling a class as a function": we are instantiating the logger with **new Logger(...)**.

**Comment by JordanBelford** [comment]
Same here. Works for node 6.7.0 but fails on 6.8.0
Here's a oneliner to reproduce, assumes le_node is installed.
```
node -e "let Logger = require('le_node'); let le = new Logger({token: '00000000-0000-0000-0000-000000000000'})"
```
Output:
```
/Users/jordan.belford/test/node_modules/babel-runtime/helpers/classCallCheck.js:7
    throw new TypeError("Cannot call a class as a function");
    ^
```

**Comment by JordanBelford** [comment]
It's a node and babel issue, see https://github.com/nodejs/node/pull/8834#issuecomment-253640692

There will likely be a 6.8.1 release soon to address this, so I don't think any changes to le_node are needed.

**Comment by larsthorup** [comment]
This issue is fixed in Node 6.8.1 which is out now.

**Comment by perhallstroem** [comment]
Works for me!
